vendor/models/offer.py

```diff
@@ -49,6 +63,8 @@ class Offer(CreateUpdateModelBase):
 
     objects = models.Manager()
     on_site = CurrentSiteManager()
+    is_available = AvailableManager()
```

Comment:
this manager should just be "available".  Make sure to update any migrations or other code using this.